grails-app/jobs/org/pih/warehouse/jobs/RefreshOrderSummaryJob.groovy

```diff
@@ -21,7 +21,7 @@ class RefreshOrderSummaryJob {
         if (JobUtils.shouldExecute(RefreshOrderSummaryJob)) {
             def startTime = System.currentTimeMillis()
             log.info("Refreshing order summary: " + context.mergedJobDataMap)
-            orderService.refreshOrderSummary()
+            orderSummaryService.refreshOrderSummary()
```

Comment:
do you know why this job is needed? It seems the refreshOrderSummary method just takes a copy of the view, deletes it, recreates it, and re-inserts the copy. I'm not sure what this is even achieving?

Reply:
It is needed to be able to trigger rebuild of entire order summary **materialized** view (`order_summary_mv` table)

Reply:
I suppose I just don't understand how the materialized views work and why they need to be dropped entirely and recreated in this job. Does it get out of sync with order_summary somehow?

Anyways, this isn't relevant to the PR itself so we can move on.